src/sections/chat/ChatSidebarSection.tsx

```diff
@@ -119,90 +119,97 @@ const ChatSidebarSection = ({ showCreateGroupModal, setSelectedChatId }) => {
-              active={activeTab == 0 ? true : false}
-              background="transparent"
-              hoverBackground="transparent"
-              color={theme.default.color}
-              flex="1"
-              zIndex="1"
-              padding="10px 10px 20px 10px"
-              onClick={() => {
-                setActiveTab(0);
-              }}
-            >
-              <SpanV2
-                fontSize="16px"
-                fontWeight="400"
-                color={activeTab === 0 ? GLOBALS.COLORS.PRIMARY_PINK : 'inherit'}
-              >
-                Chats
-              </SpanV2>
-            </TabButton>
-
-            <TabButton
-              active={activeTab == 1 ? true : false}
-              background="transparent"
-              hoverBackground="transparent"
-              color={theme.default.color}
-              zIndex="1"
-              flex="1"
-              padding="10px 10px 20px 10px"
-              onClick={() => {
-                setActiveTab(1);
-              }}
-            >
-              <ItemHV2
-                alignItems="center"
-                // ref={containerRef}
+      <ItemVV2
+        justifyContent="flex-start"
+        padding="20px 10px 0px 20px"
+        width="100%"
+        flex="0"
+        zIndex="1"
+      >
+        {/* Render Chats and Requests */}
+        {activeTab !== 3 && (
+          <ItemVV2
+            flex="initial"
+            width="100%"
+            padding="0px 0px 10px 0px"
+            minHeight="72px"
+          >
+            <ItemHV2>
+              {/* Set active and onCLick to customize tab */}
+              <TabButton
+                active={activeTab == 0 ? true : false}
+                background="transparent"
+                hoverBackground="transparent"
+                color={theme.default.color}
+                flex="1"
+                zIndex="1"
+                padding="10px 10px 20px 10px"
+                onClick={() => {
+                  setActiveTab(0);
+                }}
               >
                 <SpanV2
-                  flex="initial"
                   fontSize="16px"
                   fontWeight="400"
-                  color={activeTab === 1 ? GLOBALS.COLORS.PRIMARY_PINK : 'inherit'}
-                  margin="0px 4px"
+                  color={activeTab === 0 ? GLOBALS.COLORS.PRIMARY_PINK : 'inherit'}
+                  minHeight="20px"
                 >
-                  Requests
+                  Chats
                 </SpanV2>
-                {numberOfChatReqs === -1 || requestLoadingData?.loading ? (
-                  <LoaderSpinner
-                    type={LOADER_TYPE.SEAMLESS}
-                    width="auto"
-                    spinnerSize={20}
-                    spinnerColor={GLOBALS.COLORS.PRIMARY_PINK}
-                  />
-                ) : numberOfChatReqs > 0 ? (
+              </TabButton>
+
+              <TabButton
+                active={activeTab == 1 ? true : false}
```

Comment:
```suggestion
                active={!!activeTab}
```

Reply:
Not a boolean, but will go with what you suggest